.gitignore

```diff
@@ -24,3 +24,4 @@ deployments/localhost
 typings
 forge-cache
 out
+.env.bak
```

Comment:
.env.*

Reply:
would also cover `.env.example`, which should not be ignored